bevy_asset_loader_derive/src/assets.rs

```diff
@@ -94,6 +108,40 @@ impl AssetField {
                     asset_server.get_handle(#asset_path)
                 },)
             }
+            AssetField::Image(image) => {
+                let field_ident = image.field_ident.clone();
+                let asset_path = image.asset_path.clone();
+                match image.sampler {
```

Comment:
I think this code can be deduplicated by only returning the line that changes based on the sampler from the match.